rollup-interface/src/state_machine/da.rs

```diff
@@ -179,6 +179,8 @@ pub trait BlobReaderTrait: Serialize + DeserializeOwned + Send + Sync + 'static
 
 /// Trait with collection of trait bounds for a block hash.
 pub trait BlockHashTrait:
+    // TODO: Can we replace `Into<[u8; 32]>` with std::hash::Hash 
```

Comment:
Or, I can use `[u8; 32]` as Key in hashmap, so BlockHash does not need to implement Hash